packages/utilities/src/initials.ts

```diff
@@ -61,7 +67,7 @@ export function getInitials(displayName: string | undefined | null, isRtl: boole
 
   displayName = cleanupDisplayName(displayName);
 
-  if (UNSUPPORTED_TEXT_REGEX.test(displayName)) {
+  if (UNSUPPORTED_TEXT_REGEX.test(displayName) || PHONENUMBER_REGEX.test(displayName)) {
```

Comment:
Lets just add a small comment here explaining what we are doing

Reply:
Done.